tests/terraform_compliance/steps/its_key_value_test_cases/nested__list_of_bool.json

```diff
@@ -0,0 +1,27 @@
+{
+    "stash": [
+        {
+            "id": "a",
+            "values": [
+                {
+                    "id": "a.a",
+                    "foo": [true, false]
+                },
+                {
+                    "id": "a.b",
+                    "foo": [false, false]
+                },
+                {
+                    "id": "a.c",
+                    "not_foo": [true, false]
+                },
+                {
+                    "id": "a.d",
+                    "foo": [false, true]
+                }
+            ]
+        }
+    ],
+    "args": ["foo", true],
+    "expected_ids": []
```

Comment:
We don't support lists inside nested objects, should we?

Reply:
Same as above, since we dont have a strict expected structure due to different implementation of different terraform providers, lets focus on passing functional tests.